analysis/webservice/webmodel/NexusExecutionResults.py

```diff
@@ -53,6 +54,9 @@ def construct_done(status, created, completed, execution_id, params, host):
         params,
         host
     )
+    # Add stats to body
+    job_body['totalPrimaryMatched'] = num_primary_matched
+    job_body['averageSecondaryMatched'] = round(num_secondary_matched/num_primary_matched)
```

Comment:
I'm curious why you decided to round to nearest int here. I think returning this as a float could benefit by allowing the user to compute the number of secondaries / total number of points more accurately.

Reply:
I'm open to your idea of not rounding. I added rounding because I wasn't sure how valuable that extra info would be and I think it's nice that both values are integers representing a count. Maybe we can see if the team has a strong opinion?

Reply:
I think it may be worth bringing up. Ultimately, it's the total number of points that has the greatest affect on fetch time + size, so I feel an accurate way to determine that would be helpful for particularly large jobs / large tolerances.

Reply:
I'm also curious if the users would find the raw count of secondaries useful. We could add that to the output directly if that's something they might find useful

Reply:
Couldn't hurt IMO